### DISCUSSION TITLE
Api reamins in fetching state

### DISCUSSION THREAD

**Discussion by sumeet030990** [TanStack/query#3592] [General]
Hello, I am new to react-query 
I am facing a issue i.e. After update is complete, If i reopen the page, my queries remain in fetching state causing browser to hang
Attaching screenshot of the same

![image](https://user-images.githubusercontent.com/48065274/166969190-e57005e3-cf5b-4f70-abfc-f133ed4cdeb5.png)

**Comment by TkDodo** [comment]
Can you try to reproduce this in codesandbox please?
